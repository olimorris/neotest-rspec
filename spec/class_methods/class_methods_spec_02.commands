:9
:lua require("neotest").run.run()
