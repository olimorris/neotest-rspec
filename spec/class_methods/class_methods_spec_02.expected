./SPEC/CLASS_METHODS/CLASS_METHODS_SPEC.RB@@PASSED@@handles the "." method naming convention
